GDJS/GDJS/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -27,14 +27,8 @@ BaseObjectExtension::BaseObjectExtension() {
   std::map<gd::String, gd::ExpressionMetadata> &objectStrExpressions =
       GetAllStrExpressionsForObject("");
 
-  objectActions["MettreX"]
-      .SetFunctionName("setX")
-      .SetGetter("getX")
-      .SetIncludeFile("runtimeobject.js");
```

Comment:
Why is it not necessary anymore to add the `setIncludedFile`?

Reply:
RuntimeObject is always exported as games can't run without it.